ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml

```diff
@@ -81,7 +81,7 @@
                 metadata:
                   generateName: operator-hosted-pipeline-run
                 spec:
-                  timeout: "2h"
+                  timeout: "1.5h"
```

Comment:
This is changing the overall timeout of the hosted pipeline which should stay at `2h`. Please define the timeout in the release pipeline trigger instead.